engine-standalone-storage/src/sync/types.rs

```diff
@@ -126,9 +140,31 @@ struct BorshableTransactionMessage<'a> {
     pub transaction: BorshableTransactionKind<'a>,
 }
 
+#[derive(BorshDeserialize, BorshSerialize)]
+struct BorshableTransactionMessageV2<'a> {
+    /// Hash of the block which included this transaction
+    pub block_hash: [u8; 32],
+    /// Receipt ID of the receipt that was actually executed on NEAR
+    pub near_receipt_id: [u8; 32],
+    /// If multiple Aurora transactions are included in the same block,
+    /// this index gives the order in which they should be executed.
+    pub position: u16,
+    /// True if the transaction executed successfully on the blockchain, false otherwise.
+    pub succeeded: bool,
+    /// NEAR account that signed the transaction
+    pub signer: Cow<'a, AccountId>,
+    /// NEAR account that called the Aurora engine contract
+    pub caller: Cow<'a, AccountId>,
+    /// Amount of NEAR token attached to the transaction
+    pub attached_near: u128,
+    /// Details of the transaction that was executed
+    pub transaction: BorshableTransactionKind<'a>,
+    pub promise_data: Cow<'a, Vec<Option<Vec<u8>>>>,
```

Comment:
Should we copy-paste the same comment on this field on `TransactionMessage`? Or maybe removing all other comments and adding a meta-comment pointing to `TransactionMessage`?

Reply:
Removed field comments and made one comment on the `BorshableTransactionMessage` itself in https://github.com/aurora-is-near/aurora-engine/pull/575/commits/43874feb9c8e553e064a400da0edbfcecc39cea2